src/vs/workbench/browser/parts/titlebar/titlebarPart.ts

```diff
@@ -250,6 +262,40 @@ export class TitlebarPart extends Part implements ITitleService {
 			}
 		});
 
+		if (!isMacintosh) {
```

Comment:
@sbatten there seems to be quite a bit of duplication between this code and `onDidChangeMaximized`. I suggest to move everything into `onDidChangeMaximized` and then call it from here with the initial value to prevent that.